### DISCUSSION TITLE
Displaying different API versions

### DISCUSSION THREAD

**Discussion by topher515** [scalar/scalar#2157] [Q&A]
We're considering switching to Scalar from Redocly, but we're hoping to support multiple "versions" for our API docs. We are open to different approaches for representing different version information, but simplest is probably just a dropdown box at the top of the page allowing the viewer to select a version. (In which case the versions would conceptually just be whole additional openapi spec documents that live alongside each other.)

Does functionality like this exist in Scalar?

**Comment by hanspagel** [comment]
Currently not supported! We plan to add support for multiple specifications (#1851), that might help. For now, you can consider to add a simple header/dropdown that you add above the actual API reference, or depending on the integration render multiple URLs for different specifications.

If it’s just about some minor differences, you can also set `deprecated: true` to deprecated operations, but that might not cover your case.
